### DISCUSSION TITLE
Observation doesn't match webpage

### DISCUSSION THREAD

**Discussion by thischarles** [weather-gov/api#373] [Q&A]
**Describe the bug**
I am trying to get the current temperature using the observations endpoint for KMSP, but it isn't getting updated information. The timestamp says it's from `2020-02-07T15:53:00+00:00`. The current date is February 10, so it doesn't seem like it's been updated in days. The temperature keeps reporting it as `-3.2999999999999545`°C when the website reports it as -12°C (at the time of this writing).

**To Reproduce**
Webpage: https://forecast.weather.gov/MapClick.php?lat=44.88306&lon=-93.22889

Endpoint: https://api.weather.gov/stations/KMSP/observations/
Correlation ID: `59363a37-7eba-47fe-8174-dd5f8c2448b8`

Also seen on: https://api.weather.gov/stations/KMSP/observations/latest
Correlation ID: `b2953c1e-1851-4e03-a2db-e915f366d460`

**Expected behavior**
The current temperature reported on forecast.weather.gov should match the response from the API for the observations and latest observations.

**Environment**
Tried using a JavaScript fetch request in Chrome 79.0.3945.130 and Safari 13.0.5 on the Mac, Safari on iOS 13.3.1, Insomnia 7.0.6, and the Web Service API Try It Out feature.

**Comment by StephenClouse** [comment]
Data timeliness is an outage/operational issue, not a bug; it should be reported to NCO/OMB Tech Control as described [here](https://weather-gov.github.io/api/reporting-issues).
